be/osoc/common/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from tests import *
```

Comment:
When I run `docker exec -it osoc-be python manage.py test` I get this error

`ModuleNotFoundError: No module named 'tests'`.

For me, `from . import *` seems to fix it.

Reply:
Yeah, testing also works when I remove the line.